esp/src/eclwatch/DFUQueryWidget.js

```diff
@@ -713,7 +713,7 @@ define([
                         context.downloadToList.set("disabled", true);
                     }
                 });
-                ESPUtil.goToPageUserPreference(this.workunitsGrid, "DFUQueryWidget");
+                ESPUtil.goToPageUserPreference(this.workunitsGrid, "DFUQueryWidget_GridRowsPerPage");
                 this.workunitsGrid.startup();
```

Comment:
Are we sure these are actually set _before_ the grid startup happens?

Reply:
Yes - but retVal is a `Promise` that something is going to happen in the future...

Reply:
We can wrap it in a `then` to overcome any issues?